diff --git a/examples/sample.makefile.regen b/examples/sample.makefile.regen
index 50551b2..0b36657 100755
--- a/examples/sample.makefile.regen
+++ b/examples/sample.makefile.regen
@@ -23,7 +23,7 @@ if test -z "$ASN1PDU"	\
 	exit
 fi
 
-CMDOPTS="-pdu=${ASN1PDU} ${ASN1CMDOPTS} ${ASN1MODULES}"
+CMDOPTS="-pdu=all -pdu=${ASN1PDU} -fwide-types ${ASN1CMDOPTS} ${ASN1MODULES}"
 
 if test -x ../../asn1c/asn1c ; then
   echo "Compiling ${ASN1MODULES} using local compiler"
